schema.json

```diff
@@ -1462,6 +1462,587 @@
     ]
   },
   {
+    "name": "Optimized Checkout",

```

Comment:
Yes the section should be hidden if UCO is not enabled. At the moment the condition is getting sent through `BC_SEED_DATA` https://github.com/bigcommerce-labs/ng-stencil-editor/pull/162/files#diff-607a41934429c0248a0a01f2eb030febR88